site/src/atomics/ratio.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Ratio Atomics
```

Comment:
```suggestion
title: Aspect ratio
```

The other atomic pages' titles don't include "Atomics," so this helps maintain consistency in the lefthand nav.